lib/build.js

```diff
@@ -35,12 +35,12 @@ module.exports = ({ purgeFlag }) => {
     }
   ))
 
-  const sassBuild = () => new Promise((resolve, reject) => sass.render({
+  const sassBuild = (configPath) => new Promise((resolve, reject) => sass.render({
     file: config.sourceCSS
   }, (err, result) => {
     if (err) return reject(err)
 
-    const tempFile = path.normalize(path.resolve(process.cwd(), 'temporary-tailwind-css-file.css'))
+    const tempFile = configPath ? path.normalize(path.resolve(process.cwd(),configPath.replace(/([\w\-\.]+[\-\.][\w\-\.]+\.(\w+))/,'temporary-tailwind-css-file.css')), ): path.normalize(path.resolve(process.cwd(), 'temporary-tailwind-css-file.css') )
```

Comment:
What is this regex doing? Is this fixing something about the way it previously worked, or is this specific to your incoming changes?

Reply:
Also, would you mind cleaning up some of the style inconsistencies like the spacing, etc?

Reply:
Done.